scripts/generate-openapi.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+# pylint: skip-file
+
+import argparse
+import json
+
+from dj.api.query.engine import app
```

Comment:
Should this be `dj.api.main`?

Reply:
Oh right, thanks for catching this. I think my IDE did something funny and changed this. 😨 (fixed!)